src/org/zaproxy/zap/extension/exportreport/resources/help/contents/exportreport.html

```diff
@@ -1,334 +1,377 @@
+                        </UL>
+                    <LI>
+                    <LI>
+                        <H4>Input format:</H4>
+                        <UL>
+                            <LI>Only accepts a string list with ';' delimiter</LI>
+                        </UL>
+                    </LI>
+                    <LI>
+                        <H4>Input restriction:</H4>
+                        <UL>
+                            <LI>Only accepts 't' and 'f' for each item in the list</LI>
+                        </UL>
+                    </LI>
+                    <LI>
+                        <H4>Example:</H4>
+                        <UL>
+                            <LI>-alert_details "t;t;t;t;t;t;f;f;f;f"</LI>
+                        </UL>
+                    </LI>
+                    <LI>
+                        <H4>Explanation:</H4>
+                        <UL>
+                            <LI>In the above example, only CWE ID, WASC ID, Description, Other Info, Solution and Reference Alert Details will be included in the generated report.</LI>
+                        </UL>
+                    </LI>
+                    <LI>
+                        <H4>Validation:</H4>
+                        <UL>
+                            <LI>Content is validated to be either 't' or 'f' and that all 10 items are in the list.</LI>
+                        </UL>
+                    </LI>
+                    <LI><BR></LI>
+                </UL>
+            </LI>
+            <LI><HR></LI>
+        </UL>
+
+        <UL style="list-style: none;">
+            <LI>
+                <H4>Puttin it all together! (Windows)</H4>
+                <OL>
+                    <LI>Open a command prompt</LI>
+                    <LI>Navigate to location where the zap.bat file is located</LI>
+                    <LI>execute the following:
+                        <UL style="list-style: none;"><LI>zap.bat <B>-export_report</B> "/path/to/save/report.xhtml" <B>-source_info</B> "Vulnerability Report of MyApp.com;JordanGS;Lost Souls;August 15, 2016;August 18, 2016;ZAP_D-2016-08-15;ZAP_D-2016-08-15;Lorem ipsum dolor sit amet, pri corpora ancillae adolescens in. Ea usu atomorum tincidunt, ne munere regione has. Ne sea summo tation, et sed nibh nostrum singulis. Nec causae viderer discere eu." <B>-alert_severity</B> "t;t;f;t" <B>-alert_details</B> "t;t;t;t;t;t;f;f;f;f" <B>-session</B> "/path/to/existing/session" <B>-cmd</B></LI></UL>
+                    </LI>
+                </OL>
+            </LI>
+        </UL>
+
+        <HR>
+
+		<a name="api-calls"></a> 
+        <H2><U>API Calls</U></H2>
+
+        The extension can be accessed with API calls and requires the following arguments to be passed in to generate a report.
+
+        <BR><BR>
+
+        The restrictions are the same as those for Command Line above.
+
+        <H4>Usage Example:</H4>
+        <PRE>
+            <CODE>
+    Map<String, String> map = new HashMap<String, String>();
```

Comment:
It should use the html entities for `<` and `>` (otherwise they are not rendered).